cypress/e2e/licenses/license-lifecycle.cy.js

```diff
@@ -23,9 +26,31 @@ describe('License lifecycle', () => {
     type: 'Local'
   };
 
+  let typeCreated = false;
+
   before(() => {
     cy.login(Cypress.env('login_username'), Cypress.env('login_password'));
     cy.getAdminToken();
+    cy.getLicensesRefdataValues(refdataTypeDesc).then((refdata) => {
+      if (refdata.every(obj => obj.label !== license.type)) {
+        LicensesSettingsInteractor.setLicensesRefdataValue(refdataTypeDesc, license.type);
```

Comment:
Can we please make use of refdata value instead of label? The users could change the refdata label and this test would fail

Reply:
Ah I see... we test based on label... not thrilled about this, it feels like we should be doing refdata checks locally. I'm going to make use of a shared refdata file instead so we can test based on Label and make inferences in the code based on value

Reply:
Comments added to the code... Ended up using label still. Not 100% sure on the best way to proceed but some thoughts in place in comment form